### DISCUSSION TITLE
Lego Mindstorms Inventor Hub - sensor communication protocol

### DISCUSSION THREAD

**Discussion by ChinthaniSugandhika** [pybricks/support#663] [General]
Hi,

I am new to Lego Mindstorms Inventor Hub. I am going to use pybricks. Does any one know about the communication protocol used to communicate between sensors and the hub. Is it I2C or LEGO wireless protocol 3.0?

**Comment by dlech** [comment]
The sensors use UART to communicate with the hub. We have some documentation about it at https://github.com/pybricks/technical-info/blob/master/uart-protocol.md. Hope that helps.

**Reply by ChinthaniSugandhika** [reply]
Thank you so much for the information :)
